controller/bindings/util/util.ml

```diff
@@ -7,15 +7,17 @@ let read_from_file log_src path =
       let%lwt in_chan = Lwt_io.(open_file ~mode:Lwt_io.Input) path in
       let%lwt contents = Lwt_io.read in_chan >|= String.trim in
       let%lwt () = Lwt_io.close in_chan in
-      Some contents |> Lwt.return
+      return contents
     with
-    | Unix.Unix_error (err, fn, _) ->
+    | (Unix.Unix_error (err, fn, _)) as exn ->
       let%lwt () = Logs_lwt.err ~src:log_src
         (fun m -> m "failed to read from %s: %s" path (Unix.error_message err))
       in
-      Lwt.return None
+      fail exn
+    | exn ->
+      fail exn
```

Comment:
This exception could be hidden, in case we use `Lwt_result.catch` and `Base.Result.ok`. Why not also logging it ?

Reply:
82b06a6ddc09d4aa633590ee9fbd5f0d1e63355d